understanding/20/sign-language-prerecorded.html

```diff
@@ -11,11 +11,12 @@ <h1>Understanding Sign Language (Prerecorded)</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Videos can be accompanied by sign language</dd>
-         <dt>Author task</dt><dd>Provide sign language interpretation for audio content in existing videos</dd>
+         <dt>Goal</dt><dd>Videos can be accompanied by sign language.</dd>
+         <dt>What to do</dt><dd>Provide sign language interpretation for audio content in existing videos.</dd>
+         <dt>Why it's important</dt><dd>People who are deaf or hard of hearing have more ways to understand multimedia content.</dd> 
```

Comment:
@GreggVan suggests
```suggestion
         <dt>Why it's important</dt><dd>People who are deaf or otherwise only communicate in sign language – cannot understand content (or cannot understand it at playback speed) if it is only in captions.</dd> 
```

Reply:
I'm afraid this longer version is harder to understand. 
It also implies that all deaf people communicate in sign language, which is not true. People turning deaf at higher age have to learn sign language first, and reading might be faster for a long time.
On the other hand, people who know sign language well might prefer it for communicating. In addition, having captions as well as sign language can help to learn the language  which the user is less familiar with.
Therefore I feel the original version is more complete.
The point you made certainly is worth mentioning in the long part of the understanding document.